git.md

```diff
@@ -1,2 +1,130 @@
-# Git
+These practices:
```

Comment:
@MonikaFu, @jdhoffa, and @jacobvjk, here is the latest draft. Can you please do a quick review before we ask for everyone else's review?

At the end you'll see a "Discussion" section. I plan to move that text out from here and into the "Discussions" tab of this repo.

Reply:
@MonikaFu and @jacobvjk, just a friendly reminder of my request for your quick review before I ask for everyone else's review. Would you be able to share you opinion by the end of this week?